css/cssom/overflow-serialization.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title>CSSOM - Overlow property has different serialization than other shorthands.</title>
+    <link rel="help" href="https://drafts.csswg.org/cssom/#serialize-a-css-value">
```

Comment:
See #10014, I couldn't find a true reference for this in the spec. @Ms2ger since you added the test?